wordpress-mobile/MediaPicker-iOS
# Update MediaPicker-iOS pod

Fixes #342.

This PR:
-  updates the schema to the latest available in Xcode 11.3.1
-  defines a Swift version in the podspec

The project is written in Objective-C, so there is no project variable to define as Swift 5 at this time.

**To test:**
1. Check out this branch
2. Build and run
